apimonitor/docker-compose.yml

```
@@ -0,0 +1,69 @@
+    volumes:
+      - .:/app
+      - logs_data:/app/logs
+    env_file:
+      - .env
+    depends_on:
+      - redis
+      - worker
+
+volumes:
+  redis_data:
+  logs_data:
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)